agent/containers/images/visualizers/README.md

```diff
@@ -0,0 +1,28 @@
+# LiveMetricVisualizer
+- Build locally with podman build -t <name> -f DockerfileLive .
+- Available at quay.io/pbench/live-metric-visualizer
+- A tool available to visualize Prometheus and PCP data live during a Pbench benchmark run
+
+- Directions:
+  -   `podman pull pbench/live-metric-visualizer`
+  -   `podman run --network host pbench/live-metric-visualizer`
+
+- NOTES:
+  - Only the grafana server with preconfigured data sources for prometheus and PCP, as well as preloaded dashboards
```

Comment:
When you start with "Only" and then list several things and continue with "as well as", the "Only" becomes a bit of a misdirection. A positive and direct statement would be better.

"The container includes a Grafana server with preconfigured data sources and dashboards for Prometheus and PCP.", for example.